main.py

```diff
@@ -45,11 +45,12 @@
                    "for more verbosity.",
               type=click.IntRange(0, 3, clamp=True),
               count=True)
+@click.option("--scale-labeled-only", is_flag=True)
```

Comment:
To clarify: the value someone passes to the flag is the label the autoscaler should see on a pod in order to consider scaling up for that pod?